### ISSUE TITLE
TV signals | Opportunity to buy

### ISSUE THREAD

**Issue by thesam1545** [chrisleekr/binance-trading-bot#413]
I feel like is a good opportunity to buy based in TV signals when there is a chance from NEUTRAL to BUY or from BUY to STRONG BUY in 5 minutes or 1 hour periods or whatever.
![image](https://user-images.githubusercontent.com/90088866/152639116-66e0b812-df4d-4e5d-84e4-584db648200f.png)
Because if we buy when there is already a BUY or STRONG BUY signal, we could enter when the price is starting the down movement.
Is is to difficult to create a bot that could detect these changes for several symbols, and send a message to 3commas for instance, as a signal to buy?

**Comment by caebwallace** [comment]
I'm pretty confident with that analysis : buying when TV indicator is BUY or STRONG BUY is not the best good strategy. It's generally FOMO and indicates the top of the current trend. That feature is disabled on my side.
A better approach could be to use MACD/RSI crossing or as I'm experimenting, based on SuperTrend H1, H4 and D1.

**Comment by thesam1545** [comment]
> I'm pretty confident with that analysis : buying when TV indicator is BUY or STRONG BUY is not the best good strategy. It's generally FOMO and indicates the top of the current trend. That feature is disabled on my side.
> A better approach could be to use MACD/RSI crossing or as I'm experimenting, based on SuperTrend H1, H4 and D1.

Yes, I aggree with you, that is why I dont want to buy when STRONG BUY or BUY is already recommended from TV.
I am suggesting to buy immediataly after a change from NEUTRAL to BUY or from BUY to STRONG BUY. :) to catch the pump in the begginning.

**Comment by chrisleekr** [comment]
Closing this as a refactored v1 has been released. It's no longer relevant to the latest code.
